### PR TITLE
Add kruise game 0.8

### PR DESCRIPTION
**Pull request by lizhipeng629**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/openkruise/charts/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/openkruise/kruise-game/blob/master/CHANGELOG.md).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/openkruise/kruise/blob/master/CODE_OF_CONDUCT.md).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by chrisliu1995** [comment]
miss soft link to version/kruise-game/0.8

File: versions/kruise-game/v0.8
Comment (chrisliu1995) [inline versions/kruise-game/v0.8:1]: wrong path here

File: versions/kruise-game/0.8/README.md
[the remark this answers is not in the document]
Comment (lizhipeng629) [reply versions/kruise-game/0.8/README.md:1]: done

File: versions/kruise-game/0.8/templates/controller_manager_config.yaml
[the remark this answers is not in the document]
Comment (lizhipeng629) [reply versions/kruise-game/0.8/templates/controller_manager_config.yaml:1]: done

File: versions/kruise-game/v0.8
Comment (lizhipeng629) [reply versions/kruise-game/v0.8:1]: corrected already

**Pull request merged**